Example/Vouched/MainViewController.swift

```diff
@@ -14,4 +15,14 @@ class MainViewController: UIViewController {
         self.navigationController?.navigationBar.isHidden = true
     }
     
```

Comment:
This is how we set that conditional value - in the main view, when you click on the new liveness session button, this controller tells the FaceViewController that it is a selfie verification job, so it can post to a different endpoint